src/rgw/rgw_file.cc

```diff
@@ -377,7 +377,7 @@ namespace rgw {
   } /* RGWLibFS::unlink */
 
   int RGWLibFS::rename(RGWFileHandle* src_fh, RGWFileHandle* dst_fh,
-		       const char *_src_name, const char *_dst_name)
+		       const char *_src_name, const char *_dst_name, const char* acl="")
```

Comment:
Can we define the default value of parameter `acl` in function `rename` declaration in rgw_file.h? It should not be here.